app/views/news_tematica/news_tematicas/show.html.haml

```diff
@@ -1,9 +1,10 @@
-:sass
-  .cuentaclics
-    background: #FF3
-    color: black
-    padding: 4px
-    margin-right: 8px
+:css
+  .cuentaclics {
+    background: #FF3;
```

Comment:
Just googled and read 2 good reasons for this:

- Lowercase is easier to read for humans. EVERY TEXT IN UPPERCASE TAKES ADDITIONAL BRAINPOWER TO READ.

- The only app that ever produced uppercase hex colors is PhotoShop -> If PhotoShop does it, it should be really bad  ;-P